oc_mirror.sh

```diff
@@ -81,3 +78,74 @@ EOF
    popd
 
 }
+
+function set_full_imageset() {
+
+   # Create imageset with all images from releaseImage
+   imageset=$1
+
+   latest_release=$(oc-mirror list releases --channel candidate-${OPENSHIFT_RELEASE_STREAM}| tail -n1)
+
+   cat > "${imageset}" << EOF
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+archiveSize: 16
+storageConfig:
+  local:
+    path: metadata
+mirror:
+  platform:
+    architectures:
+      - "amd64"
+    channels:
+      - name: candidate-${OPENSHIFT_RELEASE_STREAM}
+        minVersion: $latest_release
+        maxVersion: $latest_release
+        type: ocp
+  additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+EOF
+
+}
+
+# Use the oc-mirror command to generate a tar file of the release image
+function mirror_to_file() {
```

Comment:
I'd probably merge this function with the previous one

Reply:
Its split this way to allow the file to be generated separate from the publish so we could push the file to a node and publish it to the registry there.